webapp/app/components/Main.test.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Main } from './Main'; // Undecorated version.
+import { keys } from 'lodash/fp';
+import { Main, mapDispatchToProps, mapStateToProps } from './Main'; // Undecorated version.
+import initialState from '../reducers/initialState';
 
 function setup(props) {
   return shallow(<Main {...props} />);
 }
 
 describe('<Main /> component', () => {
   it('renders itself', () => {
+    // Arrange
     const wrapper = setup({
       actions: {},
       usersActions: {}
     });
 
+    // Assert
     expect(wrapper.find('Switch')).toHaveLength(1);
     expect(wrapper.find('Route')).toHaveLength(3);
   });
+
+  describe('mapToProps functions', () => {
+    it('should return the state of auth module', () => {
+      // Act
+      const props = mapStateToProps(initialState);
+
+      // Assert
+      expect(props.isAuthenticated).toBe(false);
+    });
+
+    it('should return the binded actions ', () => {
+      // Arrange
+      const dispatch = () => {};
+      const expectedActions = [
+        'loginRequest',
+        'loginSuccess',
+        'loginError',
+        'login'
+      ];
+      // Act
```

Comment:
Please remove the commented out code, if comments are to be commited, we should make sure they serve a purpose, i.e. to explain a non obvious solution, as a general rule we avoid obvious comments to not pollute the code.

Reply:
I think is a good practice to keep tests split in three parts, see [ArrangeActAssert](http://wiki.c2.com/?ArrangeActAssert), if this project will be used as startup point for other developer this comments shall remain to keep tests patter organized

@mravinale thougths?

Reply:
Both have good arguments, but grouping in "arrange act and assert" helps a lot on reading, but if we agree to do this we should be consistent and update all tests.

personally I like more to have comments regarding "arrange act and assert", but is your choice as a team